image_client/botany_importer.py

```diff
@@ -59,8 +56,9 @@ def process_barcode(self, barcode, filepath_list):
             self.logger.debug(f"No barcode; skipping")
             return
         self.logger.debug(f"Barcode: {barcode}")
-        sql = f"select collectionobjectid  from collectionobject where catalognumber={barcode}"
+        sql = f"select CollectionObjectID from casbotany.collectionobject where CatalogNumber={barcode};"
```

Comment:
It looks like you've made it so we're being explicit about the database here. I think that's going to generalize poorly - if someone else reuses this code( possible) or we enx up generalizing it for another collection (likely) this will be a problem. We also have to be consistent and change >every< reference, so best to skip this and remove all "casbotany." from sql statements.